package/boot/uboot-mvebu/Makefile

```diff
@@ -45,6 +52,7 @@ endef
 define Build/InstallDev
 	$(INSTALL_DIR) $(STAGING_DIR_IMAGE)
 	$(CP) $(PKG_BUILD_DIR)/$(UBOOT_IMAGE) $(STAGING_DIR_IMAGE)/$(BUILD_VARIANT)-u-boot-spl.kwb
+	$(CP) $(PKG_BUILD_DIR)/$(UBOOT_IMAGE) $(STAGING_DIR_IMAGE)/$(BUILD_VARIANT)-u-boot.bin
```

Comment:
Won't this break if one file doesn't exist?
Anyway, you can just use `$(BUILD_VARIANT)-$(UBOOT_IMAGE)` instead, see my 0001-fixup.patch in the comments

Reply:
@dhewg it didn't break for missing the _u-boot-spl.kwb_, but your solution looks much better so I've updated the commit to use _$(BUILD_VARIANT)-$(UBOOT_IMAGE)_.